compiler/src/iree/compiler/Dialect/Flow/Transforms/RegionOpUtils.cpp

```diff
@@ -503,6 +503,133 @@ Flow::wrapOpInDispatchRegion(RewriterBase &rewriter, Operation *op) {
   return newRegionOp;
 }
 
+/// Returns true if the operation is a matmul or matmul-like op.
+bool Flow::isMatmulLikeOp(Operation *op) {
```

Comment:
If it is just this, then we dont need this method. We can just check for `isaContractionOpInterface` in the call site.

Reply:
cool cool, I thought it'd be neater to not do the casting stuff on call site. :D